src/systems/geometry.js

```diff
@@ -59,9 +59,13 @@ module.exports.System = registerSystem('geometry', {
     var cache = this.cache;
     var cacheCount = this.cacheCount;
     var geometry;
-    var hash = this.hash(data);
+    var hash;
+
+    if (data.skipCache || data.mergeTo) { return; }
+
+    hash = this.hash(data);

```

Comment:
This is still not very clean. It's to avoid the error ```TypeError: cyclic object value``` when sending a selector type property to the JSON.parse function